commons-packet/commons-packet-manager/src/main/java/io/mosip/commons/packet/keeper/PacketKeeper.java

```diff
@@ -72,6 +72,9 @@
 
 	@Value("${mosip.kernel.machineid.length}")
 	private int machineIdLength;
+	
+	@Value("${packet.manager.signature.required}")
+	private boolean isSignatureReqd;

```

Comment:
Rename this config to packetmanager.packet.signature.disable-verification and disablePacketSignatureVerification.

Use default value as false everywhere. In very rare conditions only country might chose to make this true.

Reply:
Renamed config.
added default value as false